source/core.py

```diff
@@ -120,6 +120,24 @@ class NewNVDAInstance:
 	directory: Optional[str] = None
 
 
+def restartUnsafely():
+	"""Force a new copy to start, which is useful if NVDA is frozen."""

```

Comment:
Can you make this comment a little more explicit about what "unsafely" means. IE
```suggestion
	"""Start a new copy of NVDA immediately.
	Used as a last resort, in the event of a serious error to immediately restart NVDA without running any cleanup / exit code. 
	There is no dependency on NVDA currently functioning correctly, which is In contrast with L{restart} which depends on the internal queue processing (queueHandler).
	Because none of NVDA's shutdown code is run, NVDA is likely to be left in an unclean state.
	Some examples of clean up that may be skipped.
	- Free NVDA's mutex (mutex prevents multiple NVDA instances), leaving it abandoned when this process ends.
	  - However, this situation is handled during mutex acquisition.
	- Remove icons (systray)
	- Saving settings
	"""

```